packages/app/components/Chatbot/Chatbot.tsx

```diff
@@ -49,8 +49,7 @@ export const ChatbotComponent: React.FC = () => {
   const [messages, setMessages] = useState<Message[]>([
     {
       type: 'apiMessage',
-      message:
-        'Hello, how can I assist you? I can help with anything course related.',
+      message: 'Hello, how can I assist you?',
```

Comment:
honestly kinda like the old message a little bit more since the generic 'Hello, how can I assist you' reminds me of a lot of those annoying bots on websites that I usually try to ignore (like when doing online banking or just wanting to read an article). The "I can help with anything course related" shows that it's a bit more personalised

Reply:
valid, i will make the change